### DISCUSSION TITLE
Translate the docs for PT-BR

### DISCUSSION THREAD

**Discussion by Viniciusmbc** [freshframework/fresh#341] [Ideas]
Hey, guys. I intend to translate documents into pt-br. Someone help me?

**Comment by fisiofaz** [comment]
posso ajudar

**Reply by Viniciusmbc** [reply]
Ok, vamos organizar então uma divisão, cada um pega uma parte e depois cada um revisa o que outro fez, pode ser?

**Reply by fisiofaz** [reply]
pode ser vc me passsa as partes e eu ajudo

**Comment by Viniciusmbc** [comment]
Você pode conversar comigo no discord ? Acredito que você tenha me achado
por lá no grupo da deco.cx. Daí criamos um trello simples e vamos separando
por página ou seção. tudo bem por ti?

Em qui., 26 de out. de 2023 às 22:59, Fábio André Zatta <
***@***.***> escreveu:
